clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -75,6 +75,40 @@ macro(DPCT_COMPILE_SYCL_CODE generated_files)
   DPCT_COMPILE_SYCL_CODE_IMP(sycl_device ${generated_files} ${ARGN})
 endmacro()
 
+macro (target_link_sycl_libraries target link_libraries_keyword)
```

Comment:
Pls add behavior test for  target_link_sycl_libraries in https://github.com/oneapi-src/SYCLomatic-test